DailyWeather/Model/MSKWeatherForecast.m

```diff
@@ -0,0 +1,208 @@
+//
+//  MSKWeatherForecast.m
+//  DailyWeather
+//
+//  Created by Lambda_School_Loaner_268 on 5/14/20.
+//  Copyright © 2020 Lambda, Inc. All rights reserved.
+//
+
+#import "MSKWeatherForecast.h"
+
+
+
+@implementation MSKCurrentForecast: NSObject
+-(instancetype) initWithTime:(NSNumber *)time
+                     summary:(NSString *)summary
+                        icon:(NSString *)icon
+             precipIntensity:(NSNumber *) precipintensity
+           precipProbability:(NSNumber *)precipProbability
+                 temperature:(NSNumber *)temperature
+         apparentTemperature:(NSNumber *)apparentTemperature
+                    humidity:(NSNumber *)humidity
+                    pressure:(NSNumber *)pressure
+                   windSpeed:(NSNumber *)windSpeed
+                 windBearing:(NSNumber *)windBearing
+                     uvIndex:(NSNumber *)uvIndex {
+    self = [super init];
+    if (self) {
+        _time = time;
+        _summary = summary;
+        _icon = icon;
+        _precipIntensity = precipintensity;
+        _precipProbability = precipProbability;
+        _temperature = temperature;
+        _apparentTemperature = apparentTemperature;
+        _humidity = humidity;
+        _pressure = pressure;
+        _windSpeed = windSpeed;
+        _windBearing = windBearing;
+        _uvIndex = uvIndex;
+    }
+    return self;
+}
+-(instancetype)initWithDict:(NSDictionary *)dictionary {
+    self = [super init];
+    if (self) {
+        _time = dictionary[@"time"];
+        _summary = dictionary[@"summary"];
+        _icon = dictionary[@"icon"];
+        _precipIntensity = dictionary[@"precipintensity"];
+        _precipProbability = dictionary[@"precipProbability"];
+        _temperature = dictionary[@"temperature"];
+        _apparentTemperature = dictionary[@"apparentTemperature"];
+        _humidity = dictionary[@"humidity"];
+        _pressure = dictionary[@"pressure"];
+        _windSpeed = dictionary[@"windSpeed"];
+        _windBearing = dictionary[@"windBearing"];
+        _uvIndex = dictionary[@"uvIndex"];
+    }
+    return self;
+}
+@end
+@implementation MSKDailyForecast: NSObject
```

Comment:
Probably best to split these classes up into different files? I'm actually not sure about that one, but that's how I've always seen it done; I could see it leading to unexpected issues this way potentially?